.github/workflows/build-and-push-image.yml

```diff
@@ -111,12 +114,48 @@ jobs:
 
       - name: Update Azure Container Apps Revision
         uses: azure/CLI@v1
+        id: azure
         with:
-          azcliversion: 2.40.0
+          azcliversion: 2.45.0
           inlineScript: |
             az config set extension.use_dynamic_install=yes_without_prompt
             az containerapp update \
               --name ${{ secrets.CIP_ACA_CONTAINERAPP_NAME }} \
               --resource-group ${{ secrets.CIP_ACA_RESOURCE_GROUP }} \
               --image ${{ secrets.CIP_ACR_URL }}/${{ env.DOCKER_IMAGE }}:${{ needs.set-env.outputs.release }} \
               --output none
+            ENDPOINT=$(az afd endpoint list -g ${{ secrets.CIP_ACA_RESOURCE_GROUP }} --profile-name ${{ secrets.CIP_ACA_RESOURCE_GROUP }}cdn --only-show-errors | jq -r '.[] | .hostName')
+            echo "::add-mask::$ENDPOINT"
+            echo "endpoint=https://${ENDPOINT}" >> $GITHUB_OUTPUT
+
+  cypress-tests:
+    name: Run Cypress Tests
+    if: needs.set-env.outputs.environment == "staging" || needs.set-env.outputs.environment == "dev"
```

Comment:
this might need to change to Staging and Dev
![image](https://user-images.githubusercontent.com/109524605/222392928-75523c48-7bd4-4cf3-bedf-b2125263424a.png)

Reply:
[#L37](https://github.com/DFE-Digital/prepare-academy-conversions/blob/dd70228dc3db12f349f6b04be281ef7fb1a0f98f/.github/workflows/build-and-push-image.yml#L37) will enforce lowercase for the `outputs.environment` value so we can safely compare "staging" and "dev"